Sources/Sockets/Core/Error.swift

```diff
@@ -30,6 +30,7 @@ public enum ErrorReason {
     case writeFailed
     
     case unsupportedSocketAddressFamily(Int32)
+	case unsupportedSocketFlag(Int32)
```

Comment:
unfortunately adding a case to a public enum is a breaking change. if there is an `unknown` case on this enum, we can use that. otherwise we need to make use of an existing case.